ToolBox/ZerodhaDownloader/ZerodhaDataDownloaderProgram.cs

```diff
@@ -128,7 +128,18 @@ public static void ZerodhaDataDownloader(IList<string> tickers, string market, s
             {
                 Log.Error($"ZerodhaDataDownloadManager.OnError(): Message: {err.Message} Exception: {err.InnerException}");
             }
+        }
 
+        private static IEnumerable<Historical> GetHistoryFromZerodha(Kite kite, List<uint> zerodhaTokenList, DateTime startDate, DateTime endDate, string interval)
+        {
+            IEnumerable<Historical> history = new List<Historical>();
```

Comment:
Nice!
nit `IEnumerable<Historical> history = new List<Historical>();` can directly be `var history = Enumerable.Empty<Historical>();`